test_fiscalyear.py

```diff
@@ -9,8 +9,138 @@
 US_FEDERAL = ('previous', 10, 1)
 UK_PERSONAL = ('same', 4, 6)
 
-# Default to U.S.
-fiscalyear.START_YEAR, fiscalyear.START_MONTH, fiscalyear.START_DAY = US_FEDERAL
+
+class TestCheckInt(object):
+    @pytest.mark.parametrize("value, exception", [
+        ('asdf', TypeError),
+        ("-999", TypeError),                # Shouldn't this be valid?
```

Comment:
Probably? It looks like `string.isdigit()` doesn't handle negative numbers. For a lot of the logic of this module, I tried to keep the behavior as similar as possible to the builtin `datetime` module. The builtin module doesn't handle strings of ints, so I'm not super concerned about this.
```
>>> import datetime
>>> datetime.date("2018", "2", "17")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: an integer is required (got type str)
```

Reply:
I agree that this is not really important. It would be a problem for a public function, but for an internal one it is probably OK to ignore this.